Readme updates

jonhocking opened the pull request:
added some context to the readme

Commit:
added logo file

Commit:
added some context to the readme

Commit:
further formatting

Commit:
Merge branch 'master' of github.com:theappbusiness/tab-ios-style-guide into readme-updates

Commit:
Merge branch 'master' of github.com:theappbusiness/tab-ios-style-guide into readme-updates

callumboddy commented:
hey man, yeah thanks for this. Looking good, only thing is I think style guides is two words.
thanks man

Commit:
changed styleguides to style guides

Commit:
- updated the github link to include a sneaky query param
- removed unnecessary bullet points with repeated links
- minor wording + formatting updates

Commit:
added links to NYT and GitHub for attribution

Commit:
changed waffley paragraph to bullet links

Commit:
changed search query to guide because of android…

Commit:
Merge branch 'master' of github.com:theappbusiness/tab-ios-style-guide into readme-updates

Commit:
Merge branch 'master' of github.com:theappbusiness/tab-ios-style-guide into readme-updates

shaps80 commented:
Cheeky sod! First he doesn't get the job done, then criticises it??
Hahahaha ;)
